transformations/transformations/single_column_coalesced.py

```diff
@@ -296,14 +312,23 @@ def process_kernel(self, routine):
         if self.resolve_sequence_association:
             transform_sequence_association(routine)
 
-        # Perform full source-inlining for member subroutines if so requested
+        # Perform full source-inlining for member subroutines
         if self.inline_members:
             inline_member_procedures(routine)
```

Comment:
Should this call still be changed to `inline_member_procedures(routine, allowed_aliases=(self.horizontal.index,))`?

Reply:
Ah, good catch. I have no means of testing this, but seems easy enough, so added as suggested.